src/components/views/messages/MessageEvent.tsx

```diff
@@ -74,6 +75,7 @@ const baseEvTypes = new Map<string, React.ComponentType<Partial<IBodyProps>>>([
     [M_POLL_START.altName, MPollBody],
     [M_BEACON_INFO.name, MBeaconBody],
     [M_BEACON_INFO.altName, MBeaconBody],
+    [VoiceBroadcastInfoEventType, VoiceBroadcastBody],
```

Comment:
It seems like we mostly use upper camel case for constant naming in js-sdk, but it's quite vague in the code style doc. I'm not really sure what 'general naming' is
> 9. lowerCamelCase is used for function and variable naming.
> 10. UpperCamelCase is used for general naming.

Reply:
Anyway, more a comment for the code style docs than this PR

Reply:
I am not sure either :laughing: 

If 9. mentions functions and variables then everything else is 10.?

Reply:
https://github.com/vector-im/element-web/issues/23354